README.md

```diff
@@ -38,29 +38,29 @@ Following an example of the configuration file.
 <Folder xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
   xmlns:xsd="http://www.w3.org/2001/XMLSchema" Name="Root">
   <Folders>
-    <Folder Name="Folder">
+    <Folder Name="Folder" CleanExistingItems="false">
       <Folders>
-        <Folder Name="Datasources" Hidden="true">
+        <Folder Name="Datasources" Hidden="true" CleanExistingItems="false">
           <DataSources>
             <DataSource ConnectionString="Data Source={{Server}}\{{Instance}};Initial Catalog=MyDB" Name="MyDB" Extension="SQL" CredentialRetrieval="Integrated" />
             <DataSource ConnectionString="Data Source={{Server}};Initial Catalog=MetaData" Name="MetaData" Extension="SQL" CredentialRetrieval="Store" UserName="user" Password="password" WindowsCredentials="True" />
           </DataSources>
         </Folder>
-        <Folder Name="Admin Reports" Hidden="true">
+        <Folder Name="Admin Reports" Hidden="true" CleanExistingItems="true">
           <Reports>
             <Report Name="Error Report" Hidden="true" FileName="Error Report.rdl" />
             <Report Name="Error Report for Export" Hidden="true" FileName="Error Report for Export.rdl" />
           </Reports>
           <Security>
-            <Security Name="Administrator">
+            <Security Name="Administrator" CleanExistingItems="false">
               <Roles>
                 <Role>Browser</Role>
                 <Role>Content Manager</Role>
               </Roles>
             </Security>
           </Security>
         </Folder>
-        <Folder Name="User Reports">>
+        <Folder Name="User Reports" CleanExistingItems="false">
```

Comment:
This should not be necessary, CleanExistingItems="false" is a default so it should not be explicitly specified unless true.

Reply:
Good point. The property will default to false if the value has not been set. I've removed it from the configuration examples now.

Reply:
Looks good